pkg/gateway/filter/handler/grpc/handler.go

```diff
@@ -1434,47 +1433,47 @@ func (s *server) Search(ctx context.Context, req *payload.Search_Request) (res *
 		}
 		return nil, err
 	}
-	targets = req.GetConfig().GetEgressFilters().GetTargets()
-	if targets != nil || s.DistanceFilters != nil {
-		addrs := make([]string, 0, len(targets)+len(s.DistanceFilters))
-		addrs = append(addrs, s.DistanceFilters...)
-		for _, target := range targets {
-			addrs = append(addrs, fmt.Sprintf("%s:%d", target.GetHost(), target.GetPort()))
-		}
-		c, err := s.egress.Target(ctx, addrs...)
-		if err != nil {
-			err = status.WrapWithUnavailable(
-				fmt.Sprintf(vald.SearchRPCName+" API egress filter targets %v not found", addrs),
-				err,
-				&errdetails.RequestInfo{
-					RequestId:   req.GetConfig().GetRequestId(),
-					ServingData: errdetails.Serialize(req),
-				},
-				&errdetails.BadRequest{
-					FieldViolations: []*errdetails.BadRequestFieldViolation{
-						{
-							Field:       "vectorizer targets",
-							Description: err.Error(),
+	filterConfigs = req.GetConfig().GetEgressFilters()
+	if filterConfigs != nil || s.DistanceFilters != nil {
```

Comment:
Is it OK that this variable(`s.DistanceFilters`) does not seem to be used?

cc: @kevindiu

Reply:
Maybe we can remove `s.DistanceFilters` as we are not using it anymore.

Reply:
As you pointed out, that description seems to have disappeared. 
I'm sorry, but would you mind correcting that? @kevindiu